monitor-agent/deploy.sh

```
@@ -0,0 +1,159 @@
+log_warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+log_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Check if region exists
+if [[ ! ${REGIONS[$REGION]} ]]; then
+    log_error "Unknown region: $REGION"
+    echo "Available regions: ${!REGIONS[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)